packages/bpk-component-section-header/src/BpkSectionHeader.tsx

```diff
@@ -32,7 +32,7 @@ export const SECTION_TYPES = {
 
 export type SectionType = (typeof SECTION_TYPES)[keyof typeof SECTION_TYPES];
 
-type Props = {
+export type Props = {
```

Comment:
As we are exporting this we ill probably want to import and export these in the `index.ts` file as well as the `.d.ts` file

Reply:
removed the props export, so no need for the import/export in the `index.ts` now 🙏